M1FExplained/Chapter25/Exercises/Exercise01.lean

```diff
@@ -0,0 +1,197 @@
+import Mathlib.Algebra.Group.Defs
+import Mathlib.Data.Complex.Basic
+import Mathlib.Analysis.Complex.Basic
+import Mathlib.Data.Nat.Basic
+
+abbrev complex_size_1 :=
+{ z : ℂ // ‖z‖  = 1 }
+
+instance one : One complex_size_1 := ⟨1, by exact CstarRing.norm_one⟩
+
+@[simp]
+lemma coe_one : (↑(1 : complex_size_1) : ℂ) = 1 := by rfl
+
+lemma mul_closed (a b : complex_size_1) : ‖(a : ℂ) * b‖ = 1 := by
+  rw [norm_mul, a.2, b.2, mul_one]
+
+--- part i
```

Comment:
Can you include the statement of the problem so the file is easier to understand?